README.md

```diff
@@ -26,6 +24,14 @@ This Javascript game was created for the Chingu Voyage 36. Our team wants to pro
 | Fun Facts about Typing                                   | Work In Progress |
 | Credits Screen                                           | Work In Progress |
 
+## Running the Project
```

Comment:
For future reference readme edits probably shouldn't be done in feature branches. In this instance it doesn't affect anything but this might make it harder to track changes in larger teams/projects.

Reply:
I didin't really changed the readme file on this branch. I just merge dev branch (base branch) to my feature branch after Pull request #32 was merged to dev, and as you can see here on this page  https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request "When you open a pull request, GitHub will set the base to the commit that branch references. If the branch is updated in the future, GitHub will not update the base branch's commit."